tgui/packages/tgui/interfaces/CameraViewer.tsx

```diff
@@ -0,0 +1,115 @@
+ * @license MIT
+ */
+
+import { Button, ByondUi, Flex, Section, Stack } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { capitalize } from './common/stringUtils';
+import { MinimapControllerData, MinimapMarkerData } from './MinimapController';
+
+export const CameraViewer = () => {
+  const { data, act } = useBackend<MinimapControllerData>();
+  const { title, theme, minimap_id, minimap_markers } = data;
+
+  return (
+    <Window title={title} theme={theme} width={950} height={700}>
+      <Window.Content>
+        <Stack justify="center">
+          <Stack.Item>
+            <Section
+              title="Minimap"
+              fill
+              buttons={
+                <Button
+                  icon="undo"
+                  color="green"
+                  onClick={() => act('reset_scale')}
+                >
+                  Reset Map Scale
+                </Button>
+              }
+            >
+              <Flex>
+                <Flex.Item>
+                  <ByondUi
+                    params={{
+                      id: minimap_id,
+                      type: 'map',
+                    }}
+                    style={{
+                      width: '600px',
+                      height: '600px',
+                    }}
+                  />
+                </Flex.Item>
+              </Flex>
+            </Section>
+          </Stack.Item>
+          <Stack.Item grow>
+            <Section scrollable fill title="Cameras">
+              <Flex direction="column">
+                <Flex.Item>
+                  {Object.values(minimap_markers)
+                    .sort((a, b) =>
+                      a.name.toUpperCase() < b.name.toUpperCase()
+                        ? -1
+                        : a.name.toUpperCase() > b.name.toUpperCase()
+                          ? 1
+                          : 0,
+                    )
+                    .map((marker) => (
+                      <MinimapIconMarker
+                        name={marker.name}
+                        pos={marker.pos}
+                        visible={marker.visible}
+                        can_be_deleted={marker.can_be_deleted}
+                        icon_state={marker.icon_state}
+                        index={marker.index}
+                        marker={marker.marker}
+                        key={marker.index}
+                        target_ref={marker.target_ref}
+                      />
+                    ))}
+                </Flex.Item>
+              </Flex>
+            </Section>
+          </Stack.Item>
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const MinimapIconMarker = (markerData: MinimapMarkerData) => {
+  const { act } = useBackend<MinimapControllerData>();
+  return (
+    <Stack>
+      <Stack.Item>
+        <Button
+          className="minimap-controller__buttons"
+          icon={'eye'}
```

Comment:
```suggestion
          icon="eye"
```
World's most trivial nitpick award goes to...